src/Runner.Worker/ActionRunner.cs

```diff
@@ -170,11 +170,15 @@ public async Task RunAsync()
                 }
             }
 
-            foreach (var input in userInputs)
+            // Validate inputs only for actions with action.yml
+            if (Action.Reference.Type == Pipelines.ActionSourceType.Repository)
```

Comment:
The real diff is:
```
if (Action.Reference.Type == Pipelines.ActionSourceType.Repository)
{
   existing code
}
```

Reply:
the current cause unwanted warning for `run` step.
ex: https://github.com/actions/canary/actions/runs/102763474
![image](https://user-images.githubusercontent.com/1750815/81738160-421d9a00-9467-11ea-9cc1-11c4d7438a19.png)

Reply:
cool, basically anyhting with action.yml